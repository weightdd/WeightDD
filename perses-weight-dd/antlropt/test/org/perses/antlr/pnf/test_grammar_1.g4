grammar test_grammar_1;

s
    : a
    | ~('c' | 'd')
    | b+
    ;

a
    : c b
    | d
    |
    ;

b
    : c d
    | d
    | d c b
    | d d
    ;

c
    : 'c'
    ;

d
    : 'd'
    ;