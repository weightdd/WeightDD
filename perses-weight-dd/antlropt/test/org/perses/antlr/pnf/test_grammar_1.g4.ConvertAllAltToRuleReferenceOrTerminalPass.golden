// Auto-generated by Perses AST.
// DO NOT MODIFY.
grammar test_grammar_1;



s
    : a
    | aux_rule__s_1
    | aux_rule__s_2
    ;

a
    : (c b | d)?
    ;

b
    : aux_rule__b_1
    | d
    | aux_rule__b_2
    | aux_rule__b_3
    ;

c
    : 'c'
    ;

d
    : 'd'
    ;

aux_rule__s_1
    : ~('c' | 'd')
    ;

aux_rule__s_2
    : b+
    ;

aux_rule__b_1
    : c d
    ;

aux_rule__b_2
    : d c b
    ;

aux_rule__b_3
    : d d
    ;

